src/routes/ssr/[template].json.ts

```diff
@@ -0,0 +1,108 @@
+import vm from 'vm';
+import type { RequestHandler } from '@sveltejs/kit/types';
+import type { OutputAsset, OutputChunk } from 'rollup';
+import { getCommit } from '$lib/git';
+import { build } from '$lib/rollup';
+import { getProps } from '$lib/svelte';
+import { existsSync, readFileSync } from 'fs';
+import { marked } from 'marked';
+
+type Output = {
+	html?: string;
+	js?: string;
+	css?: string;
+};
+
+const github = 'https://github.com/guardian/commercial-templates/blob';
+
+const prerender = (code: string): Output => {
```

Comment:
This function still remains a bit mystical to me, will have to spend some time looking over this when we start rebuilding more templates.